Dayol-iOS/UI/DiaryPaperModal/PaperAdd/View/AddPaperContentView+UICollectionView.swift

```diff
@@ -124,7 +123,8 @@ extension AddPaperContentView: UICollectionViewDataSource {
 extension AddPaperContentView: UICollectionViewDelegate {
 
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        // TODO: - 셀렉트 로직 추가
+        guard let cell = viewModel.cellModel(indexPath, paperStyle: currentTabType) else { return }
+        viewModel.selectCell(cell)
```

Comment:
cellModel은 cell이 아니니까 이렇게 네이밍 수정하는건 어떨까요?  

```suggestion
        guard let cellModel = viewModel.cellModel(indexPath, paperStyle: currentTabType) else { return }
        viewModel.selectCell(model: cellModel)
```

네이밍 제안이기에 작업량이 많아진다면 그냥 스킵하셔도 좋습니다~

Reply:
반영하겠습니다.